episodes/22-external.md

```diff
@@ -26,8 +20,44 @@ exercises: 25
+
+Your lesson documentation should contain the following information, which should be kept up to date.
+
+### Description of the project - `README` file
+
+`README`, `README.txt` or `README.md` is a plain text or Markdown file located in the project's root directory representing the default landing or home page for repositories in GitHub (and similar project repositories) by convention. It represents the first piece of documentation that people will see when visiting your lesson repository, hence it should concisely explain what the lesson is about and who it is for, and contain links and pointers to further information. For example, `README` should include:
+
+- **lesson title**
+- **lesson description**
+- **rendered version of the lesson** - even though this is typically included in the `About` section of the repository, it is always useful to link to the URL where the rendered lesson is available
+- **contact information** - include up-to-date email addresses or mailing lists or other details on how to get in touch with the lesson maintainers
+- **contributing information** - this is an opportunity to list what kinds of contribution are sought and how to get involved in lesson development for new contributors. You can provide more details in a separate `CONTRIBUTING` file within the repository’s root directory and link to it from `README`
+- **credits/acknowledgements** - make sure to credit those who have helped in the lesson's development or inspired it, and/or any resources/templates that you have reused. You can also link to a separate `AUTHORS.md` file within the repository’s root directory to list all the people who contributed to the lesson content (if this list starts to become too large to include in the `README` itself)
+- **citation** - it is a very good idea to specify a reference to an appropriate academic publication so others can cite the use of the lesson in their own publications and media. A common practice is to include the citation information in a separate `CITATION` file within the repository’s root directory and link to it from `README`. Citation can be contained in a plain text file (`CITATION`, `CITATION.txt`), a Markdown file (`CITATION.md`), or in the recently adopted [Citation File Format](https://github.com/citation-file-format/citation-file-format) file (`CITATION.cff`). The CFF is defined using [YAML](http://yaml.org/spec/1.2.2/) (which is another text-based format) and is now the recommended way of storing citations in GitHub. Advantages of using CFF are that GitHub will automatically show the citation information in the sidebar, making it more visible and accessible for visitors to your repository. In addition, metadata from `CITATION.cff` files will automatically be used by Zenodo if you publish your lesson/project there, and is recognisible by Zotero reference manager as well.
```

Comment:
```suggestion
- **citation** - it is a very good idea to specify a reference to an appropriate academic publication so others can cite the use of the lesson in their own publications and media. A common practice is to include the citation information in a separate `CITATION` file within the repository’s root directory and link to it from `README`. Citation can be contained in a plain text file (`CITATION`, `CITATION.txt`), a Markdown file (`CITATION.md`), or in the recently adopted [Citation File Format](https://github.com/citation-file-format/citation-file-format) (`CITATION.cff`). The CFF is defined using [YAML](http://yaml.org/spec/1.2.2/) (which we already encountered in the `config.yaml` file for the lesson website) and is now [the recommended way of storing citations in GitHub](https://github.blog/2021-08-19-enhanced-support-citations-github/). Advantages of using CFF are that GitHub will automatically show the citation information in the sidebar, making it more visible and accessible for visitors to your repository. In addition, metadata from `CITATION.cff` files will automatically be used by [Zenodo](https://zenodo.org/) if you publish your lesson/project there, and will be recognised by the Zotero reference manager as well.
```

Reply:
I think we might need to mention DOIs and what you might put as citation info before a lesson has an full journal citation? Something for a separate PR, though. I'll open an issue...

Reply:
Yes - I was wondering what to do about DOIs and where to mention them. I will modify the above point to mention that the citation can contain a reference to an appropriate publication or a DOI. Many lessons might not be 'officially' published but we should encourage them to get a DOI at least.

Reply:
Mentioned DOIs in ecd5759.